### PR TITLE
Happy New Year 2021!

### LINKED ISSUE #3896: Update license headers 2021
Use Maven parent 55 and update headers for java & C#

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Fixes #3896

### CODE REVIEW

File: analyzers/tests/SonarAnalyzer.UnitTest/Helpers/BlockIdMapTest.cs [hunk 1]
```diff
@@ -1,23 +1,22 @@
-﻿extern alias csharp;
```
Comment (andrei-epure-sonarsource) [inline analyzers/tests/SonarAnalyzer.UnitTest/Helpers/BlockIdMapTest.cs:1]: I don't know why this was being used in the first place, but it's great to see it gone 🎉

File: analyzers/src/SonarAnalyzer.CFG/License.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿                   GNU LESSER GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```
Comment (andrei-epure-sonarsource) [inline analyzers/src/SonarAnalyzer.CFG/License.txt:4]: I think this should be reverted - it's just the generic GNU Lesser GPL license

File: analyzers/src/SonarAnalyzer.Vsix/License.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿                   GNU LESSER GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```
Comment (andrei-epure-sonarsource) [inline analyzers/src/SonarAnalyzer.Vsix/License.txt:4]: revert

**Review by andrei-epure-sonarsource — CHANGES_REQUESTED** [review]
please revert the GNU license changes

**Comment by andrei-epure-sonarsource** [comment]
(and you can also clean the unused usings :) )

**Comment by pavel-mikula-sonarsource** [comment]
The unused usings are old technical debt that I don't want to address in this PR. It just popped up because of reactivation of the rule.

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**